index.js

```diff
@@ -22,9 +22,9 @@ module.exports = {
     }
   },
   included(app) {
-    let config = this.app.project.config(app.env);
+    let config = app.project.config();
 
-    this.appEnv = app.env;
+    this.appEnv = config.environment;
```

Comment:
When included as a sub-dependency, `app.env` seems to be undefined which makes the addon exclude itself from the build. Using `config.environment` seems to fix this. Not sure if this is the "correct" fix though.